vulnfeeds/cmd/cve-bulk-converter/main.go

```diff
@@ -94,9 +94,20 @@ func worker(wg *sync.WaitGroup, jobs <-chan string, outDir string, cnas []string
 		if slices.Contains(cnas, cve.Metadata.AssignerShortName) && cve.Metadata.State == "PUBLISHED" {
 			logger.Info("Processing "+string(cve.Metadata.CVEID), slog.String("cve", string(cve.Metadata.CVEID)))
 
-			if err = cvelist2osv.ConvertAndExportCVEToOSV(cve, outDir); err != nil {
-				logger.Warn("Failed to generate an OSV record for "+string(cve.Metadata.CVEID),
-					slog.String("cve", string(cve.Metadata.CVEID)), slog.Any("err", err))
+			osvFile, err := cvelist2osv.WriteOSVToFile(cve.Metadata.CVEID, outDir)
+			defer osvFile.Close()
+
+			metricsFile, err := cvelist2osv.WriteMetricToFile(cve.Metadata.CVEID, outDir)
+			defer metricsFile.Close()
+
+			cveId := cve.Metadata.CVEID
+			// Perform the conversion and export the results.
+			if err = cvelist2osv.ConvertAndExportCVEToOSV(cve, osvFile, metricsFile); err != nil {
+				logger.Warn("Failed to generate an OSV record", slog.String("cve", string(cveId)), slog.Any("err", err))
+			} else {
+				metricsFile.Close()
```

Comment:
Put these at the bottom of the loop, and remove the defer file.Close() since they are not necessary any more.